lib/internal/stream_base_commons.js

```diff
@@ -178,6 +189,36 @@ function onStreamRead(arrayBuffer) {
   }
 }
 
+function setStreamTimeout(msecs, callback) {
+  if (this.destroyed)
+    return;
+
+  // Type checking identical to timers.enroll()
+  msecs = validateTimerDuration(msecs);
+
+  // Attempt to clear an existing timer lear in both cases -
```

Comment:
I think there’s some sort of typo here that was copied over?

Reply:
You mean remove the word `lear` right ?

BTW, is it suitable to set `this.timeout` for both the `net.Socket` and `http2`